library/core/src/main/java/com/google/android/exoplayer2/text/ssa/SsaStyle.java

```diff
@@ -191,6 +197,15 @@ public static Integer parseColor(String ssaColorExpression) {
     return Color.argb(a, r, g, b);
   }
 
+  private static Integer parseFontSize(String fontSize) {
+    try {
+      return Integer.parseInt(fontSize);
```

Comment:
Is it definitely an integer? Seems it could be a float - shall we always just parse to float just in case?

Reply:
Yeah it is safer to parse to float, changed it.